ArduCopter/GCS_Mavlink.cpp

```diff
@@ -1473,6 +1479,18 @@ void GCS_MAVLINK_Copter::send_wind() const
         wind.z);
 }
 
+void GCS_MAVLINK_Copter::send_manual_control() const
+{
+    mavlink_msg_manual_control_send(
+        chan,
+        1,
```

Comment:
Needs to be the GCS id, not hard coded. In fact by default this actually would target the same autopilot that sends the message, particularly bad if you have any routing loops.